addPost.php

```diff
@@ -0,0 +1,134 @@
+<?php
+    include('include/init.php');
+
+    if(isset($_POST['postItem'])) {
+        $postingTitle = $_POST['postingTitle'];
+        $price = $_POST['price'];
+        $location = $_POST['location'];
+        $zipCode = $_POST['zipCode'];
+        $description = $_POST['description'];
+        $emailInput = $_POST['emailInput'];
+        $conditionInput = $_POST['conditionInput'];
+        $phoneNumberInput = $_POST['phoneNumberInput'];
+        $categoryInput = $_POST['categoryInput'];
+        // $imageFile = $_POST['fileToUpload']; //Currently there is no image file support atm (S3 In the Future?)
+        if (isset($_POST['deliveredInput'])) {
+            $deliveryAvailable = 'Yes';
+        } else {
+            $deliveryAvailable = 'No';
+        }
+        $verifiedUser = getUser($emailInput, $phoneNumberInput); 
+        if ($verifiedUser) {
+        $sellerId = $verifiedUser['userId'];
+        insertProduct($postingTitle, $price, $description, $sellerId, $location, $zipCode, $categoryInput, $deliveryAvailable, $conditionInput);
+        } else {
+            echo 'Please Login Before Trying to Post an Item!'; 
+            //This case should (almost) never occur... atm I plan on having users registering or logging in before making a post.
+        }
+    }
+
+?>
+<!DOCTYPE html>
+    <html lang="en">
+        <head>
+            <meta charset="UTF-8">
+            <meta http-equiv="X-UA-Compatible" content="IE=edge">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0">
+            <title>Document</title>
+            <link rel="stylesheet" href="style.css">
+
+        </head>
+        <body>
+            <div>
+                <form action="" method = 'post' enctype="multipart/form-data"> <!-- Comment out enctype if needed -->
```

Comment:
Enctype should already be set by the html `form` tag, so I think you can get rid of it and the comment.